crowdin/src/main/java/com/crowdin/platform/data/remote/BaseRepository.kt

```diff
@@ -48,6 +55,48 @@ internal abstract class BaseRepository : RemoteRepository {
         return path
     }
 
+    private fun containsLanguageMapping(
+        path: String,
+        formattedCode: String,
+        languageMapping: Map<String, Map<String, String>>?
+    ): Boolean {
+
+        languageMapping?.get(formattedCode)?.keys?.forEach { mappingKey ->
+            languageMapping[formattedCode]?.get(mappingKey)?.let {
+                if (path.contains("%$mappingKey%")) {
+                    return true
+                }
+
+                if (path.contains(LANGUAGE_NAME)) {
+                    return true
+                }
+            }
+        }
+
+        return false
+    }
+
+    private fun getMappedFilePath(
+        path: String,
+        formattedCode: String,
+        languageMapping: Map<String, Map<String, String>>?
+    ): String {
+
+        var result = path
+
+        languageMapping?.get(formattedCode)?.keys?.forEach { mappingKey ->
```

Comment:
Empty line on method start.(check other methods empty line can be added in case there is `super.` call in other cases we don't need it)
Also same here better to extract `languageMapping?.get(formattedCode)` into property.

Reply:
changed